### ISSUE TITLE
help

### ISSUE THREAD

**Issue by swethabethireddy** [muhk01/YOLOv5-Object-Counter#2]
how to give the input and how to run the file

**Comment by abuelgasimsaadeldin** [comment]
Yep, any way I could get the object detector pre-trained weights and test this out?

**Comment by abuelgasimsaadeldin** [comment]
@muhk01 Any environment setup or inference guide? Thanks!

**Comment by tientranhpnd** [comment]
hello, i want to run this project but i cant ~~\
